recipe/meta.yaml

```diff
@@ -8,57 +8,62 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ name }}_{{ version }}.tar.gz
  # url: https://files.salome-platform.org/Salome/medfile/med-{{ version }}.tar.gz # blacklisting conda forge
   url: https://github.com/Krande/med-copy/archive/refs/tags/{{ version }}.tar.gz
   sha256: {{ sha256 }}
   patches:
-    - patches/med-4.1.1-check-hdf5-parallel.diff  # [unix]
+    # Part of this patch is required to add a few system definitions which
+    # should indicate that they are using the 1.10 API not the 1.6 API
+    # https://github.com/HDFGroup/hdf5/issues/3797
+    - patches/ignore-hdf5-bounds-check-cmake.patch
+    - patches/ignore-hdf5-bounds-check-source.patch
+    - patches/disable-fortran.patch  # [win]
+    - patches/do_not_modify_windows_path.patch
+    - patches/install-in-runtime-and_archive.patch
+    # without this patch, OSX finds the system python since it is a newer version
+    # I (hmaarrfk) am about 90% sure it is because SWIG + CMake 3.12 FindPython + OSX
+    # has some bugs... so for OSX (and linux), we just fallback to the standard
+    # build process
+    - patches/cmake-modern-python.patch  # [win]
+    - patches/do-not-make-symlinks-on-windows.patch
 
 build:
-  number: 5
-#  string: aster_py{{ CONDA_PY }}_{{mpi}}_np{{CONDA_NPY}}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }} # [mpi != "nompi"]
-#  string: aster_py{{ CONDA_PY }}_seq_np{{CONDA_NPY}}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}     # [mpi == "nompi"]  
-  skip: true  # [py2k or osx or py>=312 or win]
-  script_env:
-    - PKG_DEBUG={{ environ.get('PKG_DEBUG', 'False') }}
+  number: 0
```

Comment:
I think the mistake was here, I thought we were bumping the version, but htat didn't happen, so we should rebuild with build "6"